secp256k1_test.go

```diff
@@ -0,0 +1,534 @@
+package secp256k1
+
+import (
+	"bytes"
+	"encoding/hex"
+	"math/big"
+	"math/rand"
+	"testing"
+)
+
+// TODO: Add test vectors. (even though they're included in libsecp itself)
+
+const LoopsN = 150
+
+var Secp256k1Order, _ = new(big.Int).SetString("FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141", 16)
```

Comment:
What do you think about that vs doing this?
`var Secp256k1Order = new(big.Int).SetBytes([]byte{255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 254, 186, 174, 220, 230, 175, 72, 160, 59, 191, 210, 94, 140, 208, 54, 65, 65})`

This is way longer and not readable, but it skips the need for `TestMain`

Reply:
I guess this can work. It is indeed ugly and not readible, but A) this is a test, so not as critical. and B) it's not as though the hexa was in any way more readable, just shorter.